run.py

```diff
@@ -7,5 +7,5 @@
 
 if __name__ == '__main__':
     from app.app import create_app
-    app = create_app()
+    app = create_app(os.environ['APP_SETTINGS'])
```

Comment:
`APP_SETTINGS` has been removed as an environment variable in #79 and shouldn't be used here.

I see here that I missed some references to it though, so I've also fixed those separately in #85.

Reply:
Done 👍

Reply:
Small note:
Removing it causes an error when creating the app from config, but it's solved in #85